tests/app/api/business/validators/application_validator/test_validate_recruiter.py

```diff
@@ -21,3 +21,47 @@ def test_can_get_errors_with_no_recruiter():
     errors = ApplicationValidator(application).validate_recruiter()
 
     assert len(errors) == 1
+
+
+def test_can_get_errors_with_for_recruiter():
+    application = Application(
+        data={
+            'recruiter': 'yes'
+        }
+    )
+    errors = ApplicationValidator(application).validate_recruiter()
+
+    assert len(errors) == 0
```

Comment:
Should this be testing when `data = {}` and `len(errors) == 1`?

Reply:
that exists already: def test_can_get_errors_with_no_recruiter():

Reply:
The test name doesn't really match up with the assert that there are no errors.  Not really sure what this is testing.  Do we need it?

Reply:
The name of this test is still bothering me.  If there are no errors how are we testing that we `can_get_errors_for_recruiter`?

Reply:
Renamed this test - I think it's useful to show a valid recruiter value generates no error (where an invalid one does).